cmd/minikube/cmd/config/util.go

```diff
@@ -125,6 +126,16 @@ func EnableOrDisableAddon(name string, val string) error {
 		return nil
 	}
 
+	if name == "istio" && enable {
+		minMem := 8192
+		minCpus := 4
+		memorySizeMB := pkgutil.CalculateSizeInMB(viper.GetString("memory"))
```

Comment:
would this still work if the memory is set in the previous runs ? and is loaded from the minikube config file ?

Reply:
Thanks @medyagh , From my testing, the memmory could be loaded successfully if it's set in the previous runs, as below:
```
 ✘ fengli@fenglis-mbp  ~  minikube -p istio-test start
🎉  minikube 1.6.2 is available! Download it: https://github.com/kubernetes/minikube/releases/tag/v1.6.2
💡  To disable this notice, run: 'minikube config set WantUpdateNotification false'

🙄  [istio-test] minikube v1.6.1 on Darwin 10.14.6
✨  Selecting 'virtualbox' driver from existing profile (alternates: [hyperkit])
🔄  Starting existing virtualbox VM for "istio-test" ...
^@^@^@⌛  Waiting for the host to be provisioned ...
^@🐳  Preparing Kubernetes v1.17.0 on Docker '19.03.5' ...
🚀  Launching Kubernetes ...
^@^@🏄  Done! kubectl is now configured to use "istio-test"
⚠️  /Users/fengli/google-cloud-sdk/bin/kubectl is version 1.15.4, and is incompatible with Kubernetes 1.17.0. You will need to update /Users/fengli/google-cloud-sdk/bin/kubectl or use 'minikube kubectl' to connect with this cluster
 fengli@fenglis-mbp  ~  minikube -p istio-test addons list
- addon-manager: enabled
- dashboard: disabled
- default-storageclass: enabled
- efk: disabled
- freshpod: disabled
- gvisor: disabled
- helm-tiller: disabled
- ingress: disabled
- ingress-dns: disabled
- istio: disabled
- istio-provisioner: enabled
- logviewer: disabled
- metrics-server: disabled
- nvidia-driver-installer: disabled
- nvidia-gpu-device-plugin: disabled
- registry: disabled
- registry-creds: disabled
- storage-provisioner: enabled
- storage-provisioner-gluster: disabled
 fengli@fenglis-mbp  ~  minikube -p istio-test addons enable istio
⚠️  Enable istio needs 8192 MB of memory and 4 CPUs.
✅  istio was successfully enabled
```